### PR TITLE
README: update mention on tests, add docs for options

### PR DESCRIPTION
**Pull request by tuxcanfly**
Updated `README` to add documentation for the recently added options, as well as updated notes on tests.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -34,14 +34,28 @@ const bytesWritten = await zipper.finalize()
...
+  exclude: ['/tmp/']
+})
+```
 
-Run `npm test` to run a simple test suite; this tests that no errors occur, but
-it does not test that the file is created correctly. Please manually inspect
-`test/output.zip` and ensure it matches the files in `test/fixtures`.
+`ignoreHidden`: whether to ignore hidden files (default: `false`)
+
+`exclude`: array of paths to exclude (default: `[]`)
```
Comment (dooglio) [inline README.md]: Mention that it will also exclude files if they match the pattern.

File: README.md [hunk 2]
```diff
@@ -34,14 +34,28 @@ const bytesWritten = await zipper.finalize()
...
+
+_Note_: this applies to files as well as directories.
+
+## Testing
 
-Admittedly it's a down and dirty test, but it is enough to exercise the library
-properly and it also provides a sample usage.
+Run `npm test` to run a simple test suite; this tests that no errors occur and
```
Comment (tuxcanfly) [inline README.md]: Not quite simple as it was, could possibly use a better adjective.

Comment (dooglio) [reply README.md]:
```suggestion
Run `npm test` to run the test suite. The tests insure that no errors occur and
```

File: README.md (hunk 1 above)
Comment (tuxcanfly) [reply README.md]: Done :+1:

**Pull request merged**